assets/js/components/dashboard-sharing/DashboardSharingDialog/SharingSettings.js

```diff
@@ -0,0 +1,154 @@
+const { useSelect, useDispatch } = Data;
+
+export default function SharingSettings() {
+	const breakpoint = useBreakpoint();
+	const { setValue } = useDispatch( CORE_UI );
+
+	const dialogOpen = useSelect(
+		( select ) => !! select( CORE_UI ).getValue( SETTINGS_DIALOG )
+	);
+	const haveSettingsChanged = useSelect( ( select ) =>
+		select( CORE_MODULES ).haveSharingSettingsChanged()
+	);
+	const documentationURL = useSelect( ( select ) => {
+		return select( CORE_SITE ).getDocumentationLinkURL(
+			'dashboard-sharing'
+		);
+	} );
+
+	// Rollback any temporary selections to saved values if settings have changed and modal is closed.
+	const { rollbackSharingSettings } = useDispatch( CORE_MODULES );
+	useEffect( () => {
+		if ( ! dialogOpen && haveSettingsChanged ) {
+			rollbackSharingSettings();
+		}
+	}, [ dialogOpen, haveSettingsChanged, rollbackSharingSettings ] );
+
+	const closeDialog = useCallback( () => {
+		setValue( SETTINGS_DIALOG, false );
+		setValue( EDITING_USER_ROLE_SELECT_SLUG_KEY, undefined );
+	}, [ setValue ] );
+
+	const openResetDialog = useCallback( () => {
+		closeDialog();
+		setValue( RESET_SETTINGS_DIALOG, true );
+	}, [ closeDialog, setValue ] );
+
+	return (
+		<Fragment>
+			<div
+				className="googlesitekit-dialog__back-wrapper"
+				aria-hidden={ breakpoint !== BREAKPOINT_SMALL }
+			>
+				<Button
+					aria-label={ __( 'Back', 'google-site-kit' ) }
+					className="googlesitekit-dialog__back"
+					onClick={ closeDialog }
+				>
+					<Icon icon={ arrowLeft } />
+				</Button>
+			</div>
+			<DialogContent className="googlesitekit-dialog__content">
+				<div className="googlesitekit-dialog__header">
+					<div
+						className="googlesitekit-dialog__header-icon"
+						aria-hidden={ breakpoint === BREAKPOINT_SMALL }
+					>
+						<span>
+							<ShareIcon width={ 20 } height={ 20 } />
+						</span>
+					</div>
+
+					<div className="googlesitekit-dialog__header-titles">
+						<h2 className="googlesitekit-dialog__title">
+							{ __(
+								'Dashboard sharing & permissions',
+								'google-site-kit'
+							) }
+						</h2>
+
+						<p className="googlesitekit-dialog__subtitle">
+							{ createInterpolateElement(
+								__(
+									'Share a view-only version of your Site Kit dashboard with other WordPress roles. <a>Learn more</a>',
+									'google-site-kit'
+								),
+								{
+									a: (
+										<Link
+											aria-label={ __(
+												'Learn more about dashboard sharing',
+												'google-site-kit'
+											) }
+											href={ documentationURL }
+											external
+										/>
+									),
+								}
+							) }
+						</p>
+					</div>
+				</div>
+
+				<div className="googlesitekit-dialog__main">
+					<DashboardSharingSettings />
+				</div>
+			</DialogContent>
+			<DialogFooter className="googlesitekit-dialog__footer">
+				<Footer
+					closeDialog={ closeDialog }
+					openResetDialog={ openResetDialog }
+				/>
+			</DialogFooter>{ ' ' }
```

Comment:
What is the whitespace here needed for?

```suggestion
			</DialogFooter>
```